db/seeds.rb

```diff
@@ -36,15 +36,17 @@
               zipcode: "18501",
               email: "michael.s@email.com",
               password: "WorldBestBoss",
-              role: 2)
+              role: 2,
+              merchant: bike_shop)
 merchant_employee = User.create!(name: "Dwight Schrute",
               address: "175 Beet Rd",
               city: "Scranton",
               state: "PA",
               zipcode: "18501",
               email: "dwightkschrute@email.com",
               password: "IdentityTheftIsNotAJoke",
-              role: 1)
+              role: 1,
+              merchant: bike_shop)
 admin_user = User.create!(name: "Leslie Knope",
               address: "14 Somewhere Ave",
               city: "Pawnee",
```

Comment:
It looks like we both updated the seeds file, but in different ways. You added a merchant at the end; I created the user off the merchant. They do the same thing, but whoever merges our pr's will have to decide which way we want to move forward with for continuity :)